Mac (m1) install says ERR! path /opt/homebrew/bin/tree-sitter file already exisits

### Problem description

Any suggestions on this? I installed Neovim 0.8  using brew. 


2 packages are looking for funding
  run `npm fund` for details
npm notice
npm notice New minor version of npm available! 9.2.0 -> 9.3.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.3.0
npm notice Run npm install -g npm@9.3.0 to update!
npm notice
npm ERR! code EEXIST
npm ERR! path /opt/homebrew/bin/tree-sitter
npm ERR! EEXIST: file already exists
npm ERR! File exists: /opt/homebrew/bin/tree-sitter
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.

Looks like a great product btw, keep up the good work!

### LunarVim version

Release 1.2

### Neovim version (>= 0.8.0)

NVIM v0.8.2

### Operating system/version

m1 Mac ventura 13.1

### Steps to reproduce

Running your terminal command to install and opting in for NodeJs dependencies. 

### Screenshots

_No response_

--- a/utils/installer/install.sh
+++ b/utils/installer/install.sh
@@ -39,8 +39,11 @@ declare -a __lvim_dirs=(
 
 declare -a __npm_deps=(
   "neovim"
-  "tree-sitter-cli"
 )
+# treesitter installed with brew causes conflicts #3738
+if ! command -v tree-sitter &>/dev/null; then
+	__npm_deps+=("tree-sitter-cli")
+fi
 
 declare -a __pip_deps=(
   "pynvim"

--- a/utils/installer/install.sh
+++ b/utils/installer/install.sh
@@ -42,7 +42,7 @@ declare -a __npm_deps=(
 )
 # treesitter installed with brew causes conflicts #3738
 if ! command -v tree-sitter &>/dev/null; then
-	__npm_deps+=("tree-sitter-cli")
+  __npm_deps+=("tree-sitter-cli")
 fi
 
 declare -a __pip_deps=(

